src/firefly/js/templates/lightcurve/ptf/PTFMissionOptions.js

```diff
@@ -149,17 +149,19 @@ export function ptfOnFieldUpdate(fieldKey, value) {
 }
 
 /**
- *
- * Gets the download option panel for PTF with specific file processor id 'PtfDownload'
+ *  This is specialized for PTF download.
+ *  Gets the download option panel for PTF with specific file processor id 'PtfDownload'
  * @param mission
  * @param cutoutSizeInDeg
  * @returns {XML}
  */
 export function ptfDownloaderOptPanel (mission, cutoutSizeInDeg) {
```

Comment:
Why is ptf having it's own download panel? Can't it share the default one like WISE?

Reply:
`FileGroupProcessor` is different.  Beside that, I don't see any differences.  It would not be hard to define a `download` property to LcConverterFactory reserved for download parameters.  But, it does not have to be done this way.

Reply:
i will raise a separate ticket to cleanup the code.